lib/board.rb

```diff
@@ -56,6 +56,17 @@ def play(move)
         @cells[move-1] = current_marker
     end
 
+    def to_s()
+        result = ""
+        (0..8).each do |i|
+            result << @cells[i]
+            if i % 3 == 2
+                result << "\n"
+            end
+        end
+        result
+    end
+
     def _full?
```

Comment:
I haven't seen Ruby methods starting with an underscore. Why did you add it? I'd remove it.